src/GraphControl/Control/Grapher.h

```diff
@@ -148,6 +148,104 @@ public
         Platform::String ^ ConvertToLinear(Platform::String ^ mmlString);
         void PlotGraph();
         void AnalyzeEquation(GraphControl::Equation ^ equation);
+       // We can't use the EValTrigUnitMode enum directly in as the property type because it comes from another module which doesn't expose
+        // it as a public enum class.  So the compiler doesn't recognize it as a valid type for the ABI boundary.
+        property int TrigUnitMode
+        {
+            void set(int value)
+            {
+                if (value != (int)m_solver->EvalOptions().GetTrigUnitMode())
+                {
+                    m_solver->EvalOptions().SetTrigUnitMode((Graphing::EvalTrigUnitMode)value);
```

Comment:
These APIs are not yet available correct? Could we get those changes in a PR in the impl project so that we can test this change out?

Reply:
Steph has them coming in with the build system stuff that's going in. So they will be live shortly.